app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -216,17 +216,20 @@ function AddressSearch({
           hint={t('in_person_proofing.body.location.po_search.address_search_hint')}
         />
       </ValidatedField>
-      <SpinnerButton
-        isWide
-        isBig
-        ref={spinnerButtonRef}
-        type="submit"
-        className="margin-y-5"
-        onClick={handleSearch}
-        spinOnClick={false}
-      >
-        {t('in_person_proofing.body.location.po_search.search_button')}
-      </SpinnerButton>
+      <div className="margin-y-5">
+        <SpinnerButton
+          isWide
+          isBig
+          ref={spinnerButtonRef}
+          type="submit"
+          onClick={handleSearch}
+          spinOnClick={false}
+          actionMessage={t('in_person_proofing.body.location.po_search.is_searching_message')}
```

Comment:
@allthesignals Could we make the text color the same as what the button color changes to when searching (primary-darker)?